test/lib/containerManager.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+var path = require('path');
+var request = require('request');
+var should = require('should');
+var level = require('level');
+
+var Loader = require('yaml-config-loader');
+var CM = require('../../lib/ContainerManager');
+var server = new CM();
+var loader = new Loader();
+var db = level('/tmp/db', {db: require('memdown')});
+
+const port = 9876;
+const reapedBuildId = 'a38c39ed-3bb9-44ba-9050-8a4337384d3b';
+const pendingBuildId = 'c65c469a-60a9-43bb-848c-e96242e6b610';
+const missingBuildId = 'fedcda29-cf3b-4ff3-83ce-308a632d477b';
+
```

Comment:
I also noticed that those 3 constants are not used in more than one test in this file. Your test `should return an error if the build id is invalid` at line 57 defines `badBuildId` within the context of the test itself. This is much better for readability, so I recommend defining your constants within the tests they're used.